packages/react/src/themes/types.ts

```diff
@@ -129,6 +137,39 @@ type EmphasisColorsStrict = Partial<{
 
 export type EmphasisColors = Extendable<EmphasisColorsStrict, ColorVariants>
 
+/**
+ * A type for a predefined category colors.
+ */
+type CategoryColorsStrict = Partial<{
```

Comment:
If this is supposed to be `Strict`, let's remove the `Partial` from the definition. If it is not suppose to be `Strict`, let's change the name to CategoryColors.

Reply:
Also these typings are specific to team theme, so we may move them to `packages/react/src/themes/teams/types.ts`